core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -1093,10 +1096,15 @@ public GHResponse route( GHRequest request )
         QueryResult fromQResult = qResults.get(0);
 
         double weightLimit = request.getHints().getDouble("defaultWeightLimit", defaultWeightLimit);
+        int maxVisistedNodesForRequest = request.getHints().getInt("routing.max_visited_nodes", maxVisitedNodes);
```

Comment:
problem is that we are very inconsistent with this and it has a reason: from URL or JS one expects `some_thing` from within Java one expects `someThing`. What we could do is internally convert all `someThing` into `some_thing` or migrate all remaining `someThing` to my preferred naming `some_thing` "somehow"